### ISSUE TITLE
Book 05: Sentence 103

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#57]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/f1f9914e-fb90-4129-a19f-943db3540ea1)

```
# sent_id = Book_05_-_The_Omens.txt:103
# text = Auh in jquac iuh qujchioa y, njman ic mocaquj, ca amo qualcan in viloa: aco ichtecque, anoço tetlatlacaltique, cana vtlica cate: anoço itla ipan chololoatiuh, in tecoco.
# text[norm] = Auh in ihcuac iuh quichihua y, niman ic mocaqui, ca amo cualcan in huiloa: azo ichtecque, anozo tetlatlazaltique, cana ohtlica cateh: anozo itla ipan chololoatiuh, in tecoco.
# text[orig] = Auh¶inj·quac·iuh·qujchioa·y·,·njman¶icmocaquj·,·ca·amo·qualcan¶in·viloa·:·aco·ichtecque·,·anoço¶tetlatlacaltique·,·cana·vtlica¶cate·:·anoço·itla·ipan·chololo¶atiuh·,·in·tecoco·.
# text[osp] = [...] y en este entendia el caminante, que algun peligro auja adelante, de ladrones o robadores: o que alguna otra desgracia, le auja de acontecer, yendo el camjno adelante.
# text[osp_norm] = [...] y en éste entendía el caminante, que algún peligro había adelante, de ladrones o robadores: o que alguna otra desgracia, le había de acontecer, yendo el camino adelante.
# references = A&D/180:8
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 97.14%
# relations = 85.71%
1	Auh	auh	CCONJ	_	_	10	_	_	_
2	in	in	SCONJ	_	_	5	mark	_	_
3	jquac	ihcuac	ADV	_	_	5	advmod	_	_
4	iuh	iuh	ADV	_	_	5	advmod	_	_
5	qujchioa	chihua	VERB	_	Subcat=Tran	10	advcl	_	_
6	y	y	CCONJ	_	_	10	cc	_	_
7	,	,	PUNCT	_	_	10	punct	_	_
8	njman	niman	ADV	_	_	10	advmod	_	_
9	ic	ic	ADV	_	_	10	advmod	_	_
10	mocaquj	caqui	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	_
11	,	,	PUNCT	_	_	10	punct	_	_
12	ca	cah|ca	PART	_	_	14	discourse	_	_
13	amo	amo	ADV	_	_	14	advmod	_	_
14	qualcan	cualcan	ADV	_	_	10	csubj	_	_
15	in	in	SCONJ	_	_	16	mark	_	_
16	viloa	huiloa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	14	csubj	_	_
17	:	:	PUNCT	_	_	19	punct	_	_
18	aco	azo	ADV	_	_	19	advmod	_	_
19	ichtecque	ichtecqui	NOUN	_	_	25	nsubj	_	_
20	,	,	PUNCT	_	_	19	punct	_	_
21	anoço	anozo	ADV	_	_	22	cc	_	_
22	tetlatlacaltique	tetlatlazaltiqui	NOUN	_	_	19	conj	_	_
23	,	,	PUNCT	_	_	25	punct	_	_
24	cana	cana	ADV	_	_	25	advmod	_	_
25	vtlica	ohtlica	ADV	_	_	16	parataxis	_	_
26	cate	catqui	AUX	_	_	25	cop	_	_
27	:	:	PUNCT	_	_	25	punct	_	_
28	anoço	anozo	ADV	_	_	31	cc	_	_
29	itla	tla	PRON	_	_	31	nsubj	_	_
30	ipan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	31	obl	_	_
31	chololoatiuh	chololoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Purpose=And|Subcat=Intr|Tense=Pres|VerbForm=Fin	25	conj	_	_
32	,	,	PUNCT	_	_	_	_	_	_
33	in	in	DET|SCONJ	_	_	34	_	_	_
34	tecoco	tecoco	NOUN	_	_	31	conj	_	_
35	.	.	PUNCT	_	_	31	punct	_	_
```

**Comment by ftyers** [comment]
López Austin:
> Y cuando así lo hacía, se entendía que no iban [los caminantes] con bien; quizá ladrones, quizá salteadores estuviesen por algún lugar en el camino; o quizá irían a huir de algo: los ofenderían.

**Comment by Lguyogiro** [comment]
I wonder if the _y_ we see throughout floco is potentially _īn_/ ? I was reading the D&A translations again and I noticed that it seems to consistently correspond to the use of "this" (or if not, a place where "this" could be added to the given translation without much change in meaning imo). e.g. the present sentence: _Auh in jquac iuh qujchioa y_ "And when he did this."

It seems a bit strange, especially because the tendency (per Lockhart) is for short _in_ to be written _yn_ and demonstrative to be written _in_.  But it is still worth considering as we investigate this.

**Comment by Lguyogiro** [comment]
Also, I think we decided elsewhere that _ic_ should be analyzed as a reln, so `NOUN/obl` instead of `ADV/advmod`.

**Comment by Lguyogiro** [comment]
I'm not sure what to do about _in tecoco_.

**Comment by Lguyogiro** [comment]
Otherwise this is looking good.

**Comment by ftyers** [comment]
A&D have
> And when he did this, then was heard: "This is by no means a good time for him who goeth; perhaps thieves or robbers are somewhere on the road, or something harmful will spring out at them.

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/1271136b-49eb-4562-a85a-5fe9d9e04ee3)


- *azo X, anozo Y* ... should these be `CCONJ`/`cc` ?
- *in huiloa* ... should this be like this?
- The *in tecoco* https://nahuatl.wired-humanities.org/content/tecoco, doesn't fit in either the LA or the A&D translations ...

**Comment by Lguyogiro** [comment]
> azo X, anozo Y ... should these be CCONJ/cc ?

I say yes.

> in huiloa ... should this be like this?

this looks right to me.

re _tecoco_,  I guess this is "something will spring out at them, something harmful." So maybe `dislocated(cholo..., tecoco)`? Or `parataxis`, or `appos(itla, tecoco)`. The latter makes the most sense to me but also results in the worst-looking tree.

**Comment by ftyers** [comment]
I went with `dislocated`... 

Merged in f7eb27d.
